.env.template

```diff
@@ -165,6 +165,12 @@ OPENAI_API_KEY=your-openai-api-key
 ## BROWSE_SPACY_LANGUAGE_MODEL is used to split sentences. Install additional languages via pip, and set the model name here. Example Chinese:  python -m spacy download zh_core_web_sm
 # BROWSE_SPACY_LANGUAGE_MODEL=en_core_web_sm
 
+## WINDOWS CHROME SETTINGS - user-data-dir & errors
+## Lauch Google Chrome, pick a dir, example "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" "--user-data-dir=G:\chrome-bank\selenium.driver"
+## Visit required sites, log into youtube, facebook and so. Make sure no pages ask for cookies opr other splash screens destroying the flow.
+#BROWSER_CHROME_USER_DATA_DIR="G:\\chrome-bank\\selenium.driver.python"
```

Comment:
Is this windows-specific? If so, I suggest prefixing with WIN or WINDOWS:
- WIN_BROWSER_CHROME_USER_DATA_DIR, or
- WIN_CHROME_USER_DATA_DIR

Reply:
I suggest dropping "WINDOWS" from line 168 if it's not windows specific.